app/uk/gov/hmrc/nisp/views/includes/continueWorking.scala.html

```diff
@@ -0,0 +1,78 @@
+@*
+* Copyright 2016 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import uk.gov.hmrc.nisp.models.{SPSummaryModel, SPChartModel}
+@import uk.gov.hmrc.nisp.views.formatting.{NispMoney, Time}
+@import uk.gov.hmrc.play.views.formatting.Dates
+@import uk.gov.hmrc.nisp.controllers.routes
+@import uk.gov.hmrc.nisp.utils.Constants
+
+@(spSummary: SPSummaryModel, currentChart: SPChartModel, forecastChart: SPChartModel)
+
+@if(spSummary.numberOfQualifyingYears < Constants.minimumQualifyingYearsNSP)  {
```

Comment:
In addition to the big comment, I don't think we should have the MQP logic in here, the template it needs is probably `reached`.